### ISSUE TITLE
2025-10-31: sgl-project/sglang 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#380]
# 每日更新报告（2025-10-31）

## sgl-project/sglang

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-10-31 23:35:22 | Xinyuan Tong | Feat: deepseek-ocr logits processor (#12415) |
| 2025-10-31 21:30:37 | sjtu_shenhai | [Bug fix] Fix severe memory waste issue with torch.empty pin_memory (#12266) |
| 2025-10-31 21:17:38 | Liangsheng Yin | Simplify watchdog (#12463) |
| 2025-10-31 20:54:50 | Kindyaa | fix:watchdog thread exception (#12328) |
| 2025-10-31 19:32:49 | Yuhong Guo | Enable bailing_moe to support TP=16 (#12369) |
| 2025-10-31 19:31:40 | Zaili Wang | fix: Add default value for backend in sample_mmmu_requests (#12256) |
| 2025-10-31 17:22:15 | Lianmin Zheng | Fix ci install to allow prerelease (#12449) |
| 2025-10-31 17:20:20 | Liangsheng Yin | [CI] fix possible port conflicts. (#12452) |
| 2025-10-31 16:13:34 | Lianmin Zheng | Clean up sgl kernel (#12413) |
| 2025-10-31 15:12:21 | Bruce-x-1997 | [bugfix] set is_prefill_only=false when mixed_chunk (#10889) |
| 2025-10-31 15:05:01 | Baizhou Zhang | [CI] Add more bins for 1-gpu CI test (#12422) |
| 2025-10-31 14:03:27 | Liangsheng Yin | Add trait for `BasePrefixCache` (#12436) |
| 2025-10-31 13:33:37 | b8zhong | Revert "fix llama4 kv cache layout" (#12437) |
| 2025-10-31 12:59:08 | Keyang Ru | [ci] install released version router (#12410) |
| 2025-10-31 12:51:30 | bppps | [Bugfix]: distinguish processors for deepseek_vl2 and deepseek_ocr to p… (#12384) |
| 2025-10-31 12:39:35 | elvischenv | feat: support trtllm_mha FP8 query attention kernel (#12307) |
| 2025-10-31 12:04:48 | Yuhong Guo | feat: Add Non-intrusive Tensor Dumping for Model Inference (#10566) |
| 2025-10-31 11:14:49 | b8zhong | Fix the shared expert & routed expert overlap in Llama 4 (#12405) |
| 2025-10-31 10:56:05 | Liangsheng Yin | Reserved abortion API when retracting (#12425) |
| 2025-10-31 10:20:35 | Liangsheng Yin | [CI] fix tests' time estimation (#12401) |
| 2025-10-31 09:56:51 | Johnsonms | [DeepSeekV32] Bug fix to ensure `page_table` and `result` in same type (#12300) |
| 2025-10-31 08:54:14 | Xinyuan Tong | [Refactor] tuning_fused_moe for MLLM and small refactor (#11224) |
| 2025-10-31 06:43:59 | lpc0220 | enable cudaProfilerApi for one batch benchmarking (#11116) |
| 2025-10-31 05:40:13 | Simo Lin | [router] 0.2.2 release (#12399) |
| 2025-10-31 05:37:28 | Keyang Ru | [router] web_search_preview tool basic implementation (#12290) |
| 2025-10-31 05:32:03 | Kaixi Hou | [NVIDIA] Add CI workloads for GB200 (#12242) |
| 2025-10-31 05:22:47 | popsiclexu | [Bug fix][PD Dissaggregation] fix prefill hanging issue with PP and DP Attention, (#12368) |
| 2025-10-31 03:49:31 | Lianmin Zheng | minor code sync (#12403) |
| 2025-10-31 03:45:10 | Keyang Ru | [router] Function call support for openai router Responses API (#12386) |
| 2025-10-31 03:41:21 | Lianmin Zheng | Update news section in README.md (#12409) |
| 2025-10-31 02:39:14 | Baizhou Zhang | [ci] Fix ci_install_deepep (#12375) |
| 2025-10-31 01:40:50 | Keyang Ru | [router] Fix safety_identifier missing (#12404) |
| 2025-10-31 00:47:12 | Xiaoyu Zhang | [CI Monitor] Fix ci_monitor perf analyzer bug (#12281) |
